src/sidebar/components/button.js

```diff
@@ -3,12 +3,27 @@ import propTypes from 'prop-types';
 
 import SvgIcon from '../../shared/components/svg-icon';
 
+/**
+ * @typedef ButtonProps
```

Comment:
Can you remove the comments from the prop-types docs at the bottom of the file. The JSDoc docs are now the canonical documentation and the prop-type docs are just there as long as there are components that have not yet been converted to JSDoc.